app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
To check if a key exists in a dictionary - it is better to use `in` with the dictionary instead: `<key> in vstr`

If you compare a value with `None` - it's necessary to use the `is` key-word
[`0 is None` - False; `0 == None` - True]

You can use the method `get` from a dictionary to check the value
`vstr.get(<key>, None)` - where `None` is a default value